tests/shared/core/training_data/story_reader/test_yaml_story_reader.py

```diff
@@ -204,6 +204,25 @@ async def test_is_yaml_file(file: Text, is_yaml_file: bool):
     assert YAMLStoryReader.is_stories_file(file) == is_yaml_file
 
 
+@pytest.mark.parametrize(
+    "file,keys,expected_result",
+    [
+        ("data/test_yaml_stories/stories.yml", ["stories"], True),
+        ("data/test_yaml_stories/stories.yml", ["stories", "something_else"], False),
+        (
+            "data/test_domains/default_retrieval_intents.yml",
+            ["intents", "responses"],
+            True,
+        ),
+        ("data/test_yaml_stories/rules_without_stories.yml", ["rules"], True),
+        ("data/test_yaml_stories/rules_without_stories.yml", ["stories"], False),
+        ("data/test_stories/stories.md", ["something"], False),
```

Comment:
This used to throw an error. Do we want to keep that behaviour?

Reply:
That seems to be almost impossible with the current way of checking since we do not convert to yaml anymore. We can add `is_likely_yaml_file` check here but the behaviour would not be the same anyway.